source/php/Module/views/noticeboard.blade.php

```diff
@@ -0,0 +1,25 @@
+<div class="{{ $classes }}">
+    <h4 class="box-title">{{ $postTitle }}</h4>
+
+    @if (!empty($introductoryText))
+        {!! wpautop($introductoryText) !!}
+    @endif
+
+    <ul>
+        @forelse ($postItems as $postItem)
+            <li>
+                @if ($postItem['link'])
+                    <a href="{{ $postItem['link'] }}">{{ $postItem['title'] }} {{ $postItem['meetingDate'] }}</a>
+                @else
+                    {{ $postItem['title'] }} {{ $postItem['meetingDate'] }}
+                @endif
+                Anslaget: {{ $postItem['published'] }}
+                @if ($postItem['unpubDate'])
+                    Tas ner: {{ $postItem['unpubDate'] }}
```

Comment:
Static strings should be handled with language functions __() and be written in english. These needs to reside outside .blade.php files to be indexable with poedit.